src/NServiceBus.AcceptanceTesting/ScenarioWithContext.cs

```diff
@@ -30,25 +30,28 @@ public Task<TContext> Run(TimeSpan? testExecutionTimeout = null)
             var scenarioContext = new TContext();
             contextInitializer(scenarioContext);
 
-            var runDescriptor = new RunDescriptor
-            {
-                ScenarioContext = scenarioContext
-            };
-            runDescriptor.Settings.Merge(settings);
```

Comment:
theoretically yes. I don't think we're ever reusing the settings (or that being a valid usage in the first place) but I'll switch back to the merge code for safety reasons. This can be addressed in a dedicated PR.